test/party/get-party-by-party-identifier.api-test.ts

```diff
@@ -0,0 +1,102 @@
+import { withAcbsAuthenticationApiTests } from '@ukef-test/common-tests/acbs-authentication-api-tests';
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES } from '@ukef-test/support/environment-variables';
+import { PartyGenerator } from '@ukef-test/support/generator/party-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+describe('GET /parties/{partyIdentifier}', () => {
```

Comment:
Can we please add another test case where by testing invalid `partyIdentifier`:

* 'ABCD'
* 'AB001'
* 'A%1£'
* ' '

Reply:
Understood thank you for looking into it does this mean we need to need URL encode the `partyIdentifier` first before we call ACBS?